examples/motoko_examples/minimal-counter-dapp/src/declarations/minimal_dapp/index.js

```diff
@@ -1,4 +1,4 @@
-import { Actor, HttpAgent } from '@dfinity/agent';
+import { Actor, HttpAgent } from "@dfinity/agent";
```

Comment:
First step, if you could run `npx prettier --write .`, that will clean up a lot of these files.

Reply:
Then incorporate the latest from main and push then this should be good